### ISSUE TITLE
two windows of pages, drag a page to the other window, it counts down

### ISSUE THREAD

**Issue by ghost** [DaAwesomeP/tab-counter#31]
Create two windows full of tabs

I used 16 tabs on one, and 14 on the other

to make them even I moved a tab...

and then the 14 tab counted down?? instead of saying 15

oh, and it kept counting down with more tabs added from the other page

**Comment by ghost** [comment]
(it could be that the context for the tab count came from the previous window??)

**Comment by DaAwesomeP** [comment]
Hmm. The tab move event does not appear to fire in the tab if you keep that tab in the foreground or it has a delay. If there are too many events at once then it is being debounced.

However, all of the other events seem to be working. So if you switch tabs or switch windows then the count should fix itself. I should note that the counter does not work by counting the number of tabs added or subtracted—it counts the total number in a window every time. So it should always correct itself if it changes a second or two later.

Unfortunately, I do not have any time to fix this now, but I would be happy to review a pull request!

**Comment by DaAwesomeP** [comment]
Possibly could be fixed by #24

**Comment by ghost** [comment]
right, the count would be from the previous window, that had 14, then 13, then 12 tabs, etc.
